py/packages/corpora/models.py

```diff
@@ -57,6 +57,9 @@ def get_relevant_splits(self, text: str, limit: int = 10):
         from corpora_ai.provider_loader import load_llm_provider
 
         llm = load_llm_provider()
+        # better_text = llm.get_synthetic_embedding_text(text)
```

Comment:
might be worth manipulating the input before finding the nearest neighbor ... but ... it's slow and expensive ...